plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/PlayAssetDeliveryTestHelper.java

```diff
@@ -46,4 +51,136 @@ public static Dictionary createAssetPackLocationsDictionary() {
     testDict.put("location2", innerDict2);
     return testDict;
   }
+
+  /**
+   * Creates a anonymous class, used for testing removePackSuccess(), that calls onSuccessListener
+   * the instant the listener is added.
+   */
+  public static Task<Void> createRemovePackSuccessTask() {
+    return new Task<Void>() {
+      @Override
+      public boolean isComplete() {
+        return false;
+      }
+
+      @Override
+      public boolean isSuccessful() {
+        return false;
+      }
+
+      @Override
+      public Void getResult() {
+        return null;
+      }
+
+      @Override
+      public <X extends Throwable> Void getResult(Class<X> exceptionType) throws X {
+        return null;
+      }
+
+      @Override
+      public Exception getException() {
+        return null;
+      }
+
+      @Override
+      public Task<Void> addOnSuccessListener(OnSuccessListener<? super Void> listener) {
+        listener.onSuccess(null);
```

Comment:
Although the callbacks are asynchronous, in our mock tasks we suppose the tasks will be completed instantly in order to save time and make test logic more streamlined.